WIKI_GENERATOR_PLAN.md

```
@@ -0,0 +1,1323 @@
+- Analysis tokens: ~200k
+- Wiki generation: ~300k input + ~50k output
+- Diagrams: ~50k input + ~10k output
+- **Total Cost (Flash):** $0.20 - $0.40
+
+**Phase 4 (Embeddings):**
+- Indexing: ~$0.01 per 100k tokens
+- Search queries: negligible
+
+### Cost Calculation Examples
+
+**Example 1: Medium Repository with Gemini 2.5 Flash**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)